code/obj/item/misc_weapons.dm

```diff
@@ -788,6 +788,107 @@
 		. = ..()
 		STOP_TRACKING_CAT(TR_CAT_HUNTER_GEAR)
 
+/obj/item/knife/butterfly
+	name = "butterfly knife"
+	desc = "A small, compact butterfly knife. Concealable when folded."
+	icon = 'icons/obj/items/weapons.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	icon_state = "butterfly-knife"
+	uses_multiple_icon_states = 1
+	var/active = 0 // 0 = closed, 1 = open
```

Comment:
```suggestion
	/// FALSE = closed, TRUE = open
	var/active = FALSE
```
if it only has two states then use TRUE/FALSE instead of 1/0, also that kind of comment should probably be a docstring like so